armor.js

```diff
@@ -0,0 +1,62 @@
+// levels lookup object
+var levels = { 0: null, 1: 1, 2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8, 9: 9, 10: 10, 11: 11, 12: 12, 13: 13, 14: 14, 15: 15, 16: "PRI", 17: "DUO", 18: "TRI", 19: "TET", 20: "PEN" }
```

Comment:
`const or let > var` I used `const` b/c you won't need to reassign levels.
```suggestion
const levels = { 0: null, 1: 1, 2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8, 9: 9, 10: 10, 11: 11, 12: 12, 13: 13, 14: 14, 15: 15, 16: "PRI", 17: "DUO", 18: "TRI", 19: "TET", 20: "PEN" }
```